botocore/serialize.py

```diff
@@ -85,6 +98,11 @@ class Serializer:
     MAP_TYPE = dict
     DEFAULT_ENCODING = 'utf-8'
 
+    def __init__(self, timestamp_precision='second'):
+        if timestamp_precision is None:
+            timestamp_precision = 'second'
+        self._timestamp_precision = timestamp_precision
```

Comment:
If the value isn't `None`, we should probably have some sort of validation and raise an error if we receive something unsupported like `nanosecond`. I know the else would handle this, but might give users the impression we actually do support nanosecond precision